frontend/ui/src/pages/Inbox/Messenger/index.tsx

```diff
@@ -36,15 +36,19 @@ const Messenger = (props: ConnectedProps<typeof connector> & RouteComponentProps
 
   return (
     <section className={styles.messengerContainer}>
-      {!!conversations && (
-        <section className={styles.messengerContainerMiddlePanel}>
-          <ConversationList />
-        </section>
-      )}
-
       <Route
         path={[`${match.url}/conversations/:conversationId`, `${match.url}`]}
-        render={props => <MessengerContainer {...props} />}
+        render={props => (
+          <Fragment>
+            {!!conversations && (
+              <section className={styles.messengerContainerMiddlePanel}>
+                <ConversationList />
+              </section>
+            )}
+
+            <MessengerContainer {...props} />
+          </Fragment>
+        )}
```

Comment:
an alternative to including ConversationList in this route would be to fetch the conversation id from the pathname in ConversationList (as match.params.conversationId would not be available)

` const match: any = matchPath(ownProps.history.location.pathname, {
    path: '/inbox/conversations/:id',
  });
    currentConversationId: match && match.params.id,`